### PR TITLE
Update react scripts

### PR DESCRIPTION
**Pull request by Sartonon**
## Description :sparkles:

- Update react scripts package
- Update linter packages
- Clean up eslint config

## Issues :bug:
**[TH-1143](https://helsinkisolutionoffice.atlassian.net/browse/TH-1143):** 
**[TH-1141](https://helsinkisolutionoffice.atlassian.net/browse/TH-1141):** 

## Additional notes :spiral_notepad:

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "editor.formatOnSave": false,
+  "editor.formatOnSave": true,
```
Comment (nikomakela) [inline .vscode/settings.json:2]: my personal thought is that we should exclude editor settings from the repo. Not everybody uses vscode, so why should they have settings for it.

File: src/icons/IconMuseum.tsx [hunk 2]
```diff
@@ -17,3 +17,5 @@ export default ({ className = '' }: Props): ReactElement => (
     </g>
   </svg>
 );
+
+export default IconMuseum;
```
Comment (nikomakela) [inline src/icons/IconMuseum.tsx:21]: export default change is fine for me, but I don't understand what's the difference between these 2.

**Review by nikomakela — APPROVED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (Sartonon) [reply .vscode/settings.json:2]: Yeah but most use it and this one is pretty important configuration. No one is going to use vscoee without this config in this project.

Comment (Sartonon) [reply .vscode/settings.json:2]: But I could delete this and add some recommended stuff to README for vscode

File: src/icons/IconMuseum.tsx (hunk 2 above)
Comment (Sartonon) [reply src/icons/IconMuseum.tsx:21]: Eslint started comlaining about these after updates and named default exports are better so they are not anonymous functions.

Comment (Sartonon) [reply src/icons/IconMuseum.tsx:21]: For example Nextjs complains about this by default

Comment (Sartonon) [reply src/icons/IconMuseum.tsx:21]: with `export default () => {}` function won't have any name in callstack when for example error happens and callstack is logged.

Comment (Sartonon) [reply src/icons/IconMuseum.tsx:21]: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-anonymous-default-export.md

File: .vscode/settings.json (hunk 1 above)
Comment (Sartonon) [reply .vscode/settings.json:2]: Just checked that basically all city-of-helsinki projects have this directory included. It has all the basic setting to get vscode work nicely with our setup so I wouldn't remove it. All vscode users would need to add those settings anyway

**Pull request merged**